core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -282,8 +282,10 @@ enum class UniqueType(
 
     CostIncreasesPerCity("Cost increases by [amount] per owned city", UniqueTarget.Building),
 
+    @Deprecated("as of 4.10.17", ReplaceWith("Only available <if [buildingFilter] is constructed in all [All] cities>"))
```

Comment:
"all all cities" looks SO bad, this is a blocker for me. 
We can either have a non-cityfilter version, or find a nicer looking cityfilter, not this.

Reply:
I'll be fine with a non cityFilter version (though this does still seem to me like a spot where a cityFilter would be wanted. There's some awkwardness with city filters/conditionals and other conditionals/triggers in general). But that'll have to wait a few hours so I can sleep

Reply:
Side note: there's a general question I have on whether these functions should include an `isPuppet` check or not. Seems weird not to, given that's how the current "requires in all cities" unique works and you can't control puppets, but I can see the argument otherwise as well. On the bright side, it cleans up the text a bit (I probably should've set the depreciation message to "in all [non-[Puppet] cities`

Side-side note: should this even use the newer cityFilter filters if cityFilter is kept? Or should it use the old ones. New ones are probably easier to work with, but they do miss out on some options. Thoughts to consider

Reply:
The new ones - if we're missing options we can fill them in :)

Reply:
How about ”in all among [filter] cities”? Changes line to read as ”in all among [All] cities”

Reply:
I feel like intuitively, global checks should probably be a separate unique, but I can be willing to back down on that front